src/pages/Main/components/MainBox/MainBox.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import "./MainBox.scss";
+import FirstLow from "./../FirstLow/FirstLow";
+
+const MainBox = ({ movies }) => {
+  const [slidePx, setSlidePx] = useState(0);
+  const [btnShow, setBtnShow] = useState("none");
+  const toPrev = () => {
+    if (slidePx < -1374) {
+      setSlidePx(slidePx + 1375);
+    } else {
+      setBtnShow("none");
+    }
+  };
+  const toNext = () => {
+    if (slidePx > -1376) {
+      setSlidePx(slidePx - 1375);
+      setBtnShow("");
+    }
+  };
+  return (
+    <div className="mainBox1">
+      <p className="filmTheme1">1970's Films</p>
+      <ul className="filmList">
+        {movies.map(movie => (
+          <FirstLow
+            slide={slidePx}
+            key={movie.id}
+            id={movie.id}
+            name={movie.name}
+            date={movie.release_date}
+            url={movie.image_url}
+          />
```

Comment:
```suggestion
          <FirstLow
            slide={slidePx}
            key={movie.id}
            movie={movie}
          />
```
이렇게 한 번에 보내면 가독성 측면에서 많은 도움이 됩니다!